src/parser.rs

```diff
@@ -1097,7 +1111,7 @@ mod tests {
   test! {
     name: comment,
     text: "# foo",
-    tree: (justfile),
+    tree: (justfile ()),
```

Comment:
Let's actually include the comment in the tree, since we want to preserve them. So this would be something like:

```rust
(justfile (comment "# foo"))
```